labs/README.md

```diff
@@ -0,0 +1,16 @@
+# Experiments
+
+This directory contains experiments, work-in-progress crates, or other code and documentation, that
+do not fall under the same stability guarantees as the main crates (`matrix-sdk`,
+`matrix-sdk-crypto`, etc.).
+
+Code and documentation here may become unmaintained or outdated, and possibly removed at any time.
```

Comment:
I do want project that are here to be maintained, at least in the sense that CI-jobs ensure we don't break them unknowingly. But with a breaking CI we might also just agree that an experiment has run its course and be removed. So, let's keep this a bit more vague...

```suggestion
Lab projects might be abandoned and possibly removed at any time.
```